business/model/person.go

```diff
@@ -0,0 +1,27 @@
+package model
+
+import "time"
+
+type (
+	Person struct {
+		Name       string    `json:"name"`
+		LastName   string    `json:"lastName"`
+		DateOfBorn time.Time `json:"dateOfBorn"`
+		Document   string    `json:"document"`
+		Tipo       string    `json:"tipo"`
```

Comment:
Não misturar inglês com português 😛 
```suggestion
		Type       string    `json:"type"`
```